CHANGELOG.md

```diff
@@ -1,16 +1,21 @@
 ## 0.3.3 (Unreleased)
+
+BUG FIXES:
+
+* `azurerm_virtual_machine` - fix issue where `vhd_uri` would not be picked up in validation ([#62](https://github.com/terraform-providers/terraform-provider-azurerm/issues/62))
```

Comment:
we tend to fill this in manually, once it's merged (otherwise we get merge conflicts all over the place) - as such would it be possible to revert this file?